internal/host/host.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/thoas/go-funk"
 )
 
+const minMasterHosts = 3
```

Comment:
Can we expose minHostsNeededForInstallation instead?

Reply:
i will move `minHostsNeededForInstallation` to common otherwise we will get cycle import issue

Reply:
fixed, moved to `common/common.go` changed the name to `MinMasterHostsNeededForInstallation`